scripts/compsets/exglobal/exglobal_fcst_nems.sh

```diff
@@ -1422,11 +1438,335 @@ runSeq::
     @$coupling_interval_fast_sec
       ATM
     @
+    ATM -> AIO
```

Comment:
need to add AIO. might be better to have the IO by default in an outer loop?

Reply:
I've added AIO to run at the same frequency as IO. This run sequence is just intended to demonstrate how to use SWIO. We may refine the run sequence at a later time, when the requirements will be further defined.